notebook/nbextensions.py

```diff
@@ -11,7 +11,7 @@
 import sys
 import tarfile
 import zipfile
-from os.path import basename, join as pjoin
+from os.path import basename, join as pjoin, normpath
```

Comment:
`normpath` will remove redundant separators. It doesn't seem to have any other side effect